code/mob/living/intangible/blob_overmind/ai.dm

```diff
@@ -299,7 +299,7 @@
 					update_lists(T)
 					spread = locate(/datum/blob_ability/spread) in abilities
 					attack = locate(/datum/blob_ability/attack) in abilities
-					lipid = locate(/datum/blob_ability/build/ribosome) in abilities
+					//lipid = locate(/datum/blob_ability/build/ribosome) in abilities
```

Comment:
This seems like a bug in original code but shouldn't variable lipid contain the lipid ability?

Reply:
I'll fix this if I ever figure out how AI blobs work